library/errata_tool_product.py

```diff
@@ -194,6 +203,9 @@ def get_product(client, short_name):
     else:
         product['state_machine_rule_set'] = None
 
+    # exd_org_group
+    product['exd_org_group'] = relationships['exd_org_group']['name']
```

Comment:
Could consider using short_name here instead of name. I figured the name is slightly better for human readability.